packages/create-graphback/src/index.ts

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+import * as yargs from 'yargs';
```

Comment:
I think knowing create has only single command. It makes no sense to have any cmd support here. but it might have sense in future if we want to extend it.

Reply:
I agree with you. And because of this I needed to do this https://github.com/aerogear/graphback/pull/1717/files#diff-2358e1b515242085670a028b71d9fa9cR6 to have a default command. 

let me clean this one up.